src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.vehicles.Bulldozer;
+import core.basesyntax.vehicles.Excavator;
+import core.basesyntax.vehicles.Truck;
+
 public class MainApp {
 
+    public static void main(String[] args) {
```

Comment:
The `main` method could be more informative. Currently, it only starts and stops the machines. Consider adding print statements that indicate which type of machine is currently working.